ement-room.el

```diff
@@ -216,12 +216,18 @@ this one automatically.")
   :type 'boolean)
 
 (defcustom ement-room-url-preview-prefix (concat "  "
-                                                 (char-to-string
-                                                  (char-from-name "BOX DRAWINGS HEAVY VERTICAL"))
+                                                 (let ((char (char-from-name "BOX DRAWINGS HEAVY VERTICAL")))
+                                                   (if (char-displayable-p char)
+                                                       (char-to-string char)
+                                                     "|"))
                                                  " ")
-  "String to insert before lines in URL previews."
+  "Prefix to insert before lines in URL previews."
   :type 'string)
 
+(defcustom ement-room-url-preview-width 50
```

Comment:
This could probably be a much larger number by default, like 200-250 characters.